core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -308,10 +308,9 @@ List<ActiveCompaction> getActiveCompactions(ServerId server)
   List<ActiveCompaction> getActiveCompactions() throws AccumuloException, AccumuloSecurityException;
 
   /**
-   * Throws an exception if a tablet server can not be contacted.
+   * Throws an exception if a compactor, scan server, or tablet server can not be contacted.
    *
-   * @param tserver The tablet server address. This should be of the form
-   *        {@code <ip address>:<port>}
+   * @param tserver The server address. This should be of the form {@code <ip address>:<port>}
```

Comment:
Could deprecate this API and add a new `ping(ServerId)` method.

Reply:
Added #4964 for this

Reply:
See comment at https://github.com/apache/accumulo/pull/4964#issuecomment-2402099388. I'm thinking we might want to leave this as-is.

Reply:
Closed #4964

Reply:
I'm thinking we might want to leave this as-is. Reason being that if a user constructs a ServerId as an argument to this method, then they might expect that this method will use the type or resource group members as part of the implementation. It doesn't it just uses the `host:port`. If the user already has a ServerId, then they can pass `ServerId.toHostPortString` when calling this.

Reply:
Decided to add `ping(ServerId)` in addition to `ping(String)`, in 4c9f311